README.md

```diff
@@ -3,6 +3,6 @@ This is a backend for Python education for YunOSC writing in Python.
 Frontend is written in Vue2. A reference to Vue2 can be found [here](https://github.com/YunOSC/PyFun-Frontend).
 
 # Contributing
-If you would like to add to add more stage or lessons a you can find more information about contributing [here](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md).
+If you would like to add more stage or lessons a you can find more information about contributing [here](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md).
```

Comment:
`lessons a you can find more` <--- I don't know what did I type..
Can you help me make it more smoothly ?